modules/gke-nodepool/main.tf

```diff
@@ -219,7 +219,7 @@ resource "google_container_node_pool" "nodepool" {
       content {
```

Comment:
`iterator = taint` missing here?

Reply:
No, you get the [block name as the default iterator
](https://developer.hashicorp.com/terraform/language/expressions/dynamic-blocks#:~:text=If%20omitted%2C%20the%20name%20of%20the%20variable%20defaults%20to%20the%20label%20of%20the%20dynamic%20block)

Reply:
If you don't specify the alias for iterator, it will have the name of the "dynamic" block: https://developer.hashicorp.com/terraform/language/expressions/dynamic-blocks 

"The iterator argument (optional) sets the name of a temporary variable that represents the current element of the complex value. If omitted, the name of the variable defaults to the label of the dynamic block."